public/js/app/pages/__tests__/__snapshots__/CourseMemo.test.js.snap

```diff
@@ -8,11 +8,13 @@ exports[`Page CourseMemo renders a side menu and a page structure 1`] = `
     <section
       class="cover-page d-none d-print-block"
     >
-      <h1 />
+      <h1>
+        TEST121 undefined 7.5 credits
```

Comment:
Why is `undefined` in the name?

Reply:
f title is missing then it is showing undefined. Now title is passed as empty string